### DISCUSSION TITLE
About collaborators and the temporary Pro account while in Beta

### DISCUSSION THREAD

**Discussion by BoswachterMarc** [opengisch/QField#3208] [Q&A]
Since a few days, I see the message that I have to many 'collaborators' for a private project.
The collaborators where all disabled (they can't sync anymore). For me there is some confusion with the definition of a collaborator. See the following questions.

1. What do you call 'collaborators'? Is every editor (it are fieldworkers, not owners - see printscreen) a collaborator?
2. For the moment - during Beta phase - I am a 'Pro' User. I thought I could have more fieldworkers but then this is perhaps not the case. As I understand, a 'Pro' user can only have one fieldworker. Fieldworkeres are just as well called 'Pro' collaborators although they don't own any project. Correct?

If the above is a correct interpretation, then I need to switch to a public or organization account.
Thank you for the clarification.

![image](https://user-images.githubusercontent.com/57868975/185348225-6dbd24bb-b4db-40d9-b837-6e52a2649a65.png)
![Knipsel1](https://user-images.githubusercontent.com/57868975/185349885-51b12a88-7cb7-4633-881f-5e268083915a.PNG)

**Comment by suricactus** [comment]
A collaborator is anyone who has access to your project if it's not public. You foreectly see you can have a maximum of 1 pro collaborator.

What is recommended is to make your project owned by an organization and add the desired users as organization members and projwct collaborators.

Read more:
https://github.com/opengisch/QField/discussions/3204
https://github.com/opengisch/QField/discussions/3206
